app/utils/chains/icons.tsx

```diff
@@ -27,8 +27,8 @@ export const MoltenIcon = (size?: number): JSX.Element => {
     <Image
       src={molten}
       style={{width: `${size}px`, height: `${size}px`}}
-      className='max-w-none'
-      alt='Molten Icon'
+      className="max-w-none"
```

Comment:
weird that these switch from single to double quotes?

Reply:
The rule I went with was single quotes for everything typescript and double quotes for html. Again somewhat arbitrary choice I don't mind changing, but my editor/copilot seems to default to that and I kind of like that it helps differentiate the two since they overlap so much in tsx files.